gradle/javadoc.gradle

```diff
@@ -53,26 +53,43 @@ allprojects {
                 addBooleanOption("html5", true)
                 addBooleanOption("-allow-script-in-comments", true)
                 header = '''
-            | <script type="text/javascript">
-            | <!-- Set the location hash in the classFrame -->
-            | try {
-            |   locationHash = top.location.hash;
-            |   if (window.name == "classFrame" && locationHash != "") {
-            |     window.location.hash = locationHash;
-            |   }
-            | } catch (error) {}
-            | <!-- GA Tracking code -->
-            | if (window.name == "" || window.name == "classFrame") {
-            |     var _elqQ = _elqQ || [];
-            |     _elqQ.push(["elqSetSiteId", "413370795"]);
-            |     _elqQ.push(["elqTrackPageView"]);
-            |     (function () {
-            |     function async_load() { var s = document.createElement("script"); s.type = "text/javascript"; s.async = true; s.src = "//img03.en25.com/i/elqCfg.min.js"; var x = document.getElementsByTagName("script")[0]; x.parentNode.insertBefore(s, x); }
-            |     if (window.addEventListener) window.addEventListener("DOMContentLoaded", async_load, false);
-            |     else if (window.attachEvent) window.attachEvent("onload", async_load);
-            |     })();
-            | }
-            | </script>'''.stripMargin()
+                    | <script type="text/javascript">
+                    | function setLocationHash() {
+                    |   try {
+                    |     locationHash = top.location.hash;
+                    |     if (window.name == "classFrame" && locationHash != "") {
+                    |       window.location.hash = locationHash;
+                    |     }
+                    |   } catch (error) {}
+                    | };
+                    | function setGATrackingCode() {
+                    |   if (window.name == "" || window.name == "classFrame") {
+                    |       var _elqQ = _elqQ || [];
+                    |       _elqQ.push(["elqSetSiteId", "413370795"]);
+                    |       _elqQ.push(["elqTrackPageView"]);
+                    |       (function () {
+                    |       function async_load() { var s = document.createElement("script"); s.type = "text/javascript"; s.async = true; s.src = "//img03.en25.com/i/elqCfg.min.js"; var x = document.getElementsByTagName("script")[0]; x.parentNode.insertBefore(s, x); }
+                    |       if (window.addEventListener) window.addEventListener("DOMContentLoaded", async_load, false);
+                    |       else if (window.attachEvent) window.attachEvent("onload", async_load);
+                    |       })();
+                    |   }
+                    | };
+                    | function setSearchUrlPrefix() {
+                    |   if (typeof getURLPrefix === 'function') {
+                    |     var getURLPrefixOri = getURLPrefix;
```

Comment:
Should this be "Uri" not "Ori"?

Reply:
Ori as in Original

Reply:
Basically we wrap the original function and if it is `undefined` or returns  `"undefined/"` there is no module directory so the function returns "" - otherwise we return the original output.